### PR TITLE
Problem: (fix #61)No running aws 1-click deployment for croeseid testnet

### LINKED ISSUE #61: Problem: No running aws 1-click deployment for croeseid testnet

### PR DESCRIPTION
**Pull request by allthatjazzleo**
Solution: add aws 1-click docs

### CODE REVIEW

File: docs/getting-started/testnet-aws-1click.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+By default, it will choose the recommended instance type (**m5a.large**) for you. But you can choose other instance type as you wish.
+![](./assets/aws_instance.png)
+
+### Step 2-3. Configure instance details
+
+![](./assets/aws_config_instance.png)
+
+| Configuration           | Value                                                                                                            |
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| _Network_               | Select a existing one or default vpc if you don't have any                                                       |
```
Comment (lezzokafka) [inline docs/getting-started/testnet-aws-1click.md:32]:
```suggestion
| _Network_               | Select an existing one or default vpc if you don't have any                                                       |
```

File: docs/getting-started/testnet-aws-1click.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+### Step 2-4. Add Storage
+- Follow default setting
+
+### Step 2-5. Add Tags
+- Create Tag if needed
+  
+### Step 2-6. Configure Security Group
+By default, the image create a new security group for you allowing ssh and p2p ports.
+You can add more rules for example rpc (26657), lcd (1317), prometheus (26660), etc.
```
Comment (lezzokafka) [inline docs/getting-started/testnet-aws-1click.md:45]:
```suggestion
You can add more rules for example: rpc (26657), lcd (1317), prometheus (26660), etc.
```

File: docs/getting-started/testnet-aws-1click.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+| Deafult Port            | Description                                                                                                      |
```
Comment (lezzokafka) [inline docs/getting-started/testnet-aws-1click.md:46]:
```suggestion
| Default Port            | Description                                                                                                      |
```

**Review by lezzokafka — COMMENTED** [review]
All good, just some minor things. Thanks @allthatjazzleo  :D

**Comment by allthatjazzleo** [comment]
@lezzokafka fixed thanks!!

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**